recipes/gawk/all/test_v1_package/conanfile.py

```diff
@@ -0,0 +1,15 @@
+from conan import ConanFile
```

Comment:
For new recipes, you dont need to include test_v1_package folder anymore: https://github.com/conan-io/conan-center-index/discussions/17406